app/dashboard/migrations/0015_auto_20190308_1945.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 2.1.7 on 2019-03-08 19:45
+
+import django.contrib.postgres.fields.jsonb
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('grants', '0015_merge_20190225_1422'),
+        ('dashboard', '0014_merge_20190221_2246'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='activity',
+            name='subscription',
+            field=models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, related_name='activities', to='grants.Subscription'),
+        ),
+        migrations.AlterField(
```

Comment:
i think you can remove the rest of the `AlterField`s from this migration since they have nothing to do with the scope of hte PR

Reply:
OK